GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h

```diff
@@ -84,23 +84,22 @@ typedef void (^GIDDisconnectCompletion)(NSError *_Nullable error);
 /// @return `YES` if `GIDSignIn` handled this URL.
 - (BOOL)handleURL:(NSURL *)url;
 
-/// Checks if there is a previously authenticated user saved in keychain.
+/// Checks if there is a previous user sign-in saved in keychain.
```

Comment:
signed-in user?

Reply:
It's a little awkward, but I'm trying to be consistent with the method naming and the docs for `restorePreviousSignInWithCompletion:`.